tempest/api/workloadmgr/barbican/test_barbican.py

```diff
@@ -2340,3 +2340,143 @@ def test_5_barbican(self):
             reporting.set_test_script_status(tvaultconf.FAIL)
         finally:
             reporting.test_case_to_write()
+
+    @decorators.attr(type='workloadmgr_api')
+    def test_6_barbican(self):
+        try:
+            test_var = "tempest.api.workloadmgr.barbican.test_"
+            tests = [[test_var + "create_workload_with_encryption_CLI", 0],
+                     [test_var + "Create_Multiple_workloads_with_same_Secret_UUID", 0],
+                     [test_var + "create_workload_with_wrong_secretUUID", 0]]
+            reporting.add_test_script(tests[0][0])
+            self.kp = self.create_key_pair(tvaultconf.key_pair_name)
+            self.vm_id = self.create_vm(key_pair=self.kp)
+            self.volumes = []
+            fip = self.get_floating_ips()
+            LOG.debug("\nAvailable floating ips are {}: \n".format(fip))
+            if len(fip) < 1:
+                raise Exception("Floating ips unavailable")
+            self.set_floating_ip(fip[0], self.vm_id)
+
+            ssh = self.SshRemoteMachineConnectionWithRSAKey(fip[0])
+            self.install_qemu(ssh)
+            self.addCustomfilesOnLinuxVM(ssh, "/opt", 3)
+            md5sums_before_full = self.calculatemmd5checksum(ssh, "/opt")
```

Comment:
Do you need md5sum verification for workload creation tests ? If not remove this code. Also remove code related to floating ips.

Reply:
sure

Reply:
updated